build_ios.sh

```diff
@@ -42,23 +117,16 @@ CMAKE_OPTIONS="${CMAKE_OPTIONS} -DUNITY_ROOT_DIR=${UNITY_ROOT_DIR}"
 CMAKE_OPTIONS="${CMAKE_OPTIONS} -DFIREBASE_UNITY_BUILD_TESTS=ON"
 CMAKE_OPTIONS="${CMAKE_OPTIONS} -DFIREBASE_CPP_BUILD_STUB_TESTS=ON" # enable a stub gtest target to get abseil-cpp working.
 
-printf "#################################################################\n"
-date
-printf "Building config 'unity' on platform 'ios' with option ''.\n"
-printf "#################################################################\n"
-
-DIR=ios_unity
-
 # Display commands being run.
 set -x
 
 # Make a directory to work in (if doesn't exist)
-mkdir -p "$DIR"
+mkdir -p "$buildpath"
 
-pushd "$DIR"
+pushd "$buildpath"
 
   # Configure cmake with option value
-  cmake -DCMAKE_TOOLCHAIN_FILE=../cmake/unity_ios.cmake .. ${CMAKE_OPTIONS}
+  cmake -DCMAKE_TOOLCHAIN_FILE=${sourcepath}/cmake/unity_ios.cmake -DCMAKE_OSX_ARCHITECTURES=$SUPPORTED_ARCHITECTURES .. ${CMAKE_OPTIONS}
```

Comment:
The .. means this is still assuming that the buildpath is one directory below the sourcepath.

Reply:
the buildpath is one directory below, move it to the front to be more clear.

Reply:
My concern is that buildpath is now a setting that can be passed in, so if the caller did something like "ios/x86", it would fail, because that is now 2 directories down. Maybe just a comment on the flag would be fine as well.

Reply:
using ${sourcepath} to replace ".."